apex-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -50,6 +50,12 @@
       </includes>
     </fileSet>
     <fileSet filtered="true" encoding="UTF-8">
+      <directory>src/main/resources</directory>
+      <includes>
+        <include>**/apexlog4j.properties</include>
```

Comment:
The included log config here is for application. I am just curious, why we should not use custom filename for log4j.properties?